nbs/methods/07_vaecf.ipynb

```diff
@@ -21,7 +21,16 @@
    "cell_type": "code",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #35.        @auto_reshaping('x')
</pre><p>We should jit this function. Write <code>@partial(jax.jit, static_argnums=[0, 2])</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BirkhoffG/ReLax/pull/151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>